openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -497,6 +518,21 @@ def _verify_user(request, response, userid_in_session):
 
         if not (request_user_object_mismatch or session_user_mismatch):
             # Great! No mismatch.
+
+            try:
```

Comment:
[bug] There are a couple of earlier `return` statements in `_verify_user`, so some requests will not be logged by this code. Maybe move to right after where `_verify_user` is called, conditional on the return value?